cmake.groovy

```diff
@@ -4,7 +4,7 @@ import com.gargoylesoftware.htmlunit.html.*;
 import net.sf.json.*
 import com.gargoylesoftware.htmlunit.WebClient
 
-def baseUrl = 'http://www.cmake.org/files/'
+def baseUrl = 'https://www.cmake.org/files/'
```

Comment:
Am Donnerstag, 5. November 2015, 10:05:42 schrieb Christopher Orr:
> > @@ -4,7 +4,7 @@ import com.gargoylesoftware.htmlunit.html.*;
> > 
> >  import net.sf.json.*
> >  import com.gargoylesoftware.htmlunit.WebClient
> > 
> > -def baseUrl = 'http://www.cmake.org/files/'
> > +def baseUrl = 'https://www.cmake.org/files/'
> 
> The canonical base URL appears to be `https://cmake.org` (i.e. without
> `www.` prefix) — using that prefix (or `http://`) gets 301-redirected,
> which Jenkins currently [can't
> handle](https://issues.jenkins-ci.org/browse/JENKINS-23507).

Thanks for pointing that out, I'll fix that.

But now I#M getting
java.security.cert.CertificateException: No subject alternative DNS name 
matching cmake.org found.

Could it be an issue with the cert on cmake.org?

> openssl s_client -showcerts -connect www.cmake.org:443
CONNECTED(00000003)
depth=2 C = US, O = DigiCert Inc, OU = www.digicert.com, CN = DigiCert Global 
Root CA
verify return:1
depth=1 C = US, O = DigiCert Inc, CN = DigiCert SHA2 Secure Server CA
verify return:1
depth=0 C = US, ST = New York, L = Clifton Park, O = "Kitware, Inc.", CN = 
*.kitware.com


CN in the cert is '*.kitware.com'


-- 
Cd wrttn wtht vwls s mch trsr.